demo/checkbox-basic-demos.html

```diff
@@ -6,36 +6,131 @@
       }
     </style>
 
-    <h3>Basic Checkbox</h3>
-    <vaadin-demo-snippet id="checkbox-basic-demos-default-checkbox">
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-checkbox checked>I agree with <a href>Terms & Conditions</a></vaadin-checkbox>
+        <vaadin-checkbox value="Option" checked>Option</vaadin-checkbox>
       </template>
     </vaadin-demo-snippet>
 
-    <h3>Custom Label</h3>
-    <vaadin-demo-snippet id="checkbox-basic-demos-multiline-checkbox">
+    <h3>Basic usage with radio group</h3>
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-checkbox checked>
-          <iron-icon icon="lumo:bell" style="vertical-align: top;"></iron-icon>
-          Notifications
-          <br>
-          <small>Enable desktop notifications</small>
-        </vaadin-checkbox>
+        <vaadin-checkbox-group label="Label" theme="vertical">
+          <vaadin-checkbox value="Option one" checked>Option one</vaadin-checkbox>
```

Comment:
The values should be something else than the formatted/localized labels. Maybe "1", "2" and "3".

Reply:
Done.